test/overrides

```diff
@@ -0,0 +1,7 @@
+# Placeholder file to overwrite ci-operator variables for tests.
+# This file should be used ONLY for development. 
+# No permanent overrides should be added here. 
+# Permanent overrides should be in openshift/release repository
+#
+# example: 
+# export TEST_FOCUS="my-awesome-test-I-want-to-run"
```

Comment:
This is not a blacklist but a whitelist. I would expect us to use both, default TEST_FOCUS to conformance, and mainly use TEST_SKIP for blacklist. If you wanted to test a single test you should still be able to do it, but I'd vote for extending our verification test to fail in case you touch `TEST_FOCUS`.

Reply:
I would say fail verification if this file changes in content. As its development file, as soon as you come up with changes you need/want to do - they should be reflected in release repository. This is just to make life easier when coming up with the list/changes.

Reply:
I am fine with it, you need to add a validation script and plug it in `make verify`.

Reply:
https://github.com/openshift/openshift-azure/issues/435 will do this after, its just how I plan my work :) dont want to implement this before I have it running and working.